constants.py

```diff
@@ -685,3 +686,56 @@
 
 kVelocityControlMotorType = DCMotor.falcon500()
 kVelocityControlkV = 0.01
+
+kAngleMotorRatio = (64 / 16) * (60 / 18)
+kShootingMotorRatio = 24 / 36
+
+# change numbers later
+
+kAngleMotorCANId = 99
+kAngleMotorName = "ShooterAngleMotor"
+kAngleMotorPGain = 0.1
+kAngleMotorIGain = 0
+kAngleMotorDGain = 0
+kAngleMotorInverted = False
+
+kLeftShootingMotorCANId = 999
+kLeftShootingMotorName = "LeftShootingMotor"
+kLeftShootingMotorPIDSlot = 0
+kLeftShootingMotorPGain = 1
+kLeftShootingMotorIGain = 0
+kLeftShootingMotorDGain = 0
+kLeftShootingMotorInverted = False
+
+kRightShootingMotorCANId = 9999
+kRightShootingMotorName = "RightShootingMotor"
+kRightShootingMotorPIDSlot = 0
+kRightShootingMotorPGain = 1
+kRightShootingMotorIGain = 0
+kRightShootingMotorDGain = 0
+kRightShootingMotorInverted = False
+
+kAngleMotorMappingFunction = lambda x, y: x * y
+kLeftShootingMotorMappingFunction = lambda x, y: x * y
+kRightShootingMotorMappingFunction = lambda x, y: x * y
+kRobotAngleMappingFunction = lambda x, y: x * y
+
+kShooterAngleEncoderCANId = 1
+kShooterAngleMotorOffset = Rotation2d.fromDegrees(10.207848)
```

Comment:
whats defined as angle 0?
also is this number random? if so indicate must be changed